### PR TITLE
Add Toolkit Block

### PR DESCRIPTION
**Pull request by saugatmalla**
Test Urls: 
- Before: https://main--blog--adobecom.hlx.page/drafts/smalla/toolkit-demo
- After: https://toolkit--blog--adobecom.hlx.page/drafts/smalla/toolkit-demo

### CODE REVIEW

File: blocks/toolkit/toolkit.js [hunk 1]
```diff
@@ -0,0 +1,83 @@
...
+  }
+
+  if (!href) {
+    throw new Error('Expected link');
+  }
+
+  columns[0].innerHTML = '';
+  columns[0].append(image);
+  columns[0].classList.add('row-image');
+  columns[1].classList.add('row-content');
```
Comment (meganthecoder) [inline blocks/toolkit/toolkit.js:23]: You use these a lot so you might want to pull them out as variables

File: blocks/toolkit/toolkit.js [hunk 2]
```diff
@@ -0,0 +1,83 @@
+import { getLibs } from '../../scripts/utils.js';
+
+function decorateRow(row, createTag) {
+  const columns = Array.from(row.children);
+  const image = columns[0].querySelector('img');
+  const { href } = columns[0].querySelector('a');
+  const icon = createTag('img', { src: '/blocks/toolkit/link-icon.svg' });
```
Comment (meganthecoder) [inline blocks/toolkit/toolkit.js:7]: alt text?

File: blocks/toolkit/toolkit.js [hunk 3]
```diff
@@ -0,0 +1,83 @@
...
+    return;
+  }
+
+  if (links[0]) {
+    links[0].classList.add('con-button', 'button-s', 'outline');
+  }
+
+  if (links[1]) {
+    links[1].classList.add('con-button', 'button-s', 'blue');
+  }
```
Comment (meganthecoder) [inline blocks/toolkit/toolkit.js:51]: Nit: optional chaining
```suggestion
  links[0]?.classList.add('con-button', 'button-s', 'outline');
  links[1]?.classList.add('con-button', 'button-s', 'blue');
```

**Review by meganthecoder — APPROVED** [review]

**Review by rgclayton — APPROVED** [review]

File: blocks/toolkit/toolkit.css [hunk 4]
```diff
@@ -0,0 +1,89 @@
...
+    }
+
+    .toolkit .column :first-child {
+        margin-top: 0;
+    }
+
+    .toolkit .wrapper {
+        order: initial;
+    }
+}
```
Comment (rgclayton) [inline blocks/toolkit/toolkit.css:89]: The tabbing on this whole file looks double of all the other files. Looks like four spaces?

**Pull request merged**